app.py

```diff
@@ -0,0 +1,217 @@
+	def __init__(self, room_name, room_type):
+		self.created_rooms = {}
+		self.room_name = room_name
+		self.room_type = room_type
+
+	def create_room(self, room_name, room_type):
+		
+		if(isinstance(self.room_name, list) == True):
+			for name in self.room_name:
+				self.created_rooms[name] = self.room_type
+			return self.created_rooms
+		else:
+			self.created_rooms[self.room_name] = self.room_type
+			return self.created_rooms
+
+
+#class to create an office
+class Office(Room):
+	
+	def __init__(self):
+		self.max_occupants = 4
+		self.all_rooms = {}
+
+	def create_room(self, room_name, room_type):
+		super(Office, self).__init__(room_name, room_type)
+		self.all_rooms = super(Office, self).create_room(self.room_name, self.room_type)
+		return	self.all_rooms
+
+
+#class to create a living space
+class LivingSpace(Room):
+	
+	def __init__(self):
+		self.max_occupants = 6
+		self.all_rooms = {}
+
+	def create_room(self, room_name, room_type):
+		super(LivingSpace, self).__init__(room_name, room_type)
+		self.all_rooms = super(LivingSpace, self).create_room(self.room_name, self.room_type)
+		return self.all_rooms
+
+
+#class to create a person
+class Person(object):
+
+	def __init__(self, person_name, person_type, wants_accommodation = 'N'):
+		self.person_name = person_name
+		self.person_type = person_type
+		self.wants_accommodation = wants_accommodation
+		self.person_created = []
+
+	def add_person(self, person_name, person_type, wants_accommodation):
+		
+		if(self.wants_accommodation and self.wants_accommodation == "Y" 
+			and self.person_type == "Staff"):
+			return "Staff are not allocated living quarters."
+		elif(not self.wants_accommodation and self.person_type == "Staff" 
+			or self.wants_accommodation == "N"):
+			self.person_created = [self.person_name, self.person_type]
+		elif(not self.wants_accommodation and self.person_type == "Fellow"):
+			self.person_created = [self.person_name, self.person_type, 'N']
+		elif(self.wants_accommodation and self.person_type == "Fellow"):
+			self.person_created = [self.person_name, self.person_type, self.wants_accommodation]
+		
+		return self.person_created
+
+
+#class to create staff
+class Staff(Person):
+	
+	def __init__(self):
+		self.created_staff = []
+
+	def add_person(self, person_name, person_type, wants_accommodation = 'N'):
+		super(Staff, self).__init__(person_name, person_type, wants_accommodation)
+		self.created_staff = super(Staff, self).add_person(self.person_name, 
+			self.person_type, self.wants_accommodation)
+		return self.created_staff
+		
+
+#class to create a fellow
+class Fellow(Person):
+	
+	def __init__(self):
+		self.created_fellow = []
+
+	def add_person(self, person_name, person_type, wants_accommodation = 'N'):
+		super(Fellow, self).__init__(person_name, person_type, wants_accommodation)
+		self.created_fellow = super(Fellow, self).add_person(self.person_name, 
+			self.person_type, self.wants_accommodation)
+		return self.created_fellow
+		
+import cmd
```

Comment:
Import statements should ideally appear at the top of your file, before any class definition, unless there is a condtitional execution that needs at most an extra module that is not required anywhere else.